src/pkg/cli/agree_tos.go

```diff
@@ -21,7 +20,7 @@ func InteractiveAgreeToS(ctx context.Context, c client.FabricClient) error {
 		return nil
 	}
 
-	fmt.Println("Our latest terms of service can be found at https://defang.io/terms-service.html")
+	term.Println("Our latest terms of service can be found at https://s.defang.io/tos")
```

Comment:
was added in https://github.com/DefangLabs/s.defang.io/commit/45e28390536479f68d55b52badabd5a40ad86743